tableauserverclient/models/property_decorators.py

```diff
@@ -99,3 +106,25 @@ def validate_regex_decorator(self, value):
             return func(self, value)
         return validate_regex_decorator
     return wrapper
+
+
+def property_is_datetime(func):
```

Comment:
Even though this isn't being used right now, I left it in.  If you feel like I should delete it, I can.